### DISCUSSION TITLE
[Feature Request] Use uv, if uv is available.

### DISCUSSION THREAD

**Discussion by vedantroy** [Comfy-Org/ComfyUI-Manager#1266] [Ideas]
uv is a very fast Python package manager. It has the `uv pip` subcommand which is a drop-in replacement for pip (insofar as you're in an active venv).

Do you think it might be possible to use `uv pip` if `uv` is detected?

uv: https://docs.astral.sh/uv/

**Comment by ltdrdata** [comment]
We are already preparing to use uv.

**Comment by ltdrdata** [comment]
You can set `use_uv` in `config.ini`
https://github.com/ltdrdata/ComfyUI-Manager#config
